gsa/src/gmp/models/alert.js

```diff
@@ -129,6 +129,15 @@ class Alert extends Model {
       ret.filter = Filter.fromObject(object.filter);
     }
 
+    const methDatRepForm = ret.method.data.report_formats;
+
+    if (hasValue(methDatRepForm)) {
+      const methDatRepFormSplit = methDatRepForm.value.split(',');
```

Comment:
Btw. the backend should already provide an array here instead of a CSV list. See our new advises page for hyperion.

Reply:
Hm. OK. It currently still provides a CSV list because the report_formats method data does not have any special parsing in our fields. but maybe that can go on the list of todos.